int32 mode
float64[33] position
float64 traj_time
